metadata_plugin.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+
+import sys
+import typing
+from dataclasses import dataclass, field
+from arcaflow_plugin_sdk import plugin
+from ansible.executor.task_queue_manager import TaskQueueManager
+from ansible.parsing.dataloader import DataLoader
+from ansible.module_utils.common.collections import ImmutableDict
+from ansible.inventory.manager import InventoryManager
+from ansible.playbook.play import Play
+from ansible.vars.manager import VariableManager
+from ansible.utils.unsafe_proxy import AnsibleUnsafeText
+from ansible import context
+
+@dataclass
+class InputParams:
+    """
+    This is the data structure for the input parameters of the step defined
+    below.
+    """
+    pass
+
+
+@dataclass
+class SuccessOutput:
+    """
+    This is the output data structure for the success case.
+    """
+    metadata: typing.Any = field()
+
+
+@dataclass
+class ErrorOutput:
+    """
+    This is the output data structure in the error  case.
```

Comment:
If you come back to edit this file again, perhaps remove the extra space if you think of it.